src/collections/members/philip-obiora/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+name: Philip Obiora
+position: Contributor
+image_path: ./philip-obiora.webp
+github: Philip-21
+twitter: Philip_obiora21
+linkedin: philip-obiora-aa09401a9
+location: Lagos, Nigeria
+bio: I am a backend developer passionate in developing server side Software. An adept Gopher with an enthusiasm for Open source and continuous learning. I have high interest in Cloud computing and Kubernetes.
+status: Active
+published: true
+---
```

Comment:
Let's add the `meshery` badge to Philip's profile ✅

Reply:
Thanks I just accepted the badge 🙂